### PR TITLE
bugfix/PP-12929: Webhooks for Confirmed Transactions Should Not Be Sent After 60 Days.

### PR DESCRIPTION
**Pull request by vinothss4u**

### CODE REVIEW

File: payrexx/src/Service/PayrexxOrderService.php [hunk 1]
```diff
@@ -98,16 +102,33 @@ public function getPrestaStatusByPayrexxStatus($transactionStatus)
...
+        $refundStatusId = (int) \Configuration::get(self::PS_STATUS_REFUND);
+        $newStatusId = (int) \Configuration::get($newStatus);
+        if ($oldStatusId === $newStatusId && $newStatusId !== $refundStatusId) {
+            return false;
+        }
+        $orderFinalStatuses = [
+            (int) \Configuration::get(self::PS_STATUS_REFUND),
+            (int) \Configuration::get(self::PS_STATUS_PAYMENT),
+            (int) \Configuration::get(self::PS_STATUS_SHIPPING),
+            (int) \Configuration::get(self::PS_STATUS_DELIVERED),
```
Comment (mathis-hermann) [inline payrexx/src/Service/PayrexxOrderService.php:114]: Status Bankwire is missing. Or is removed on purpose?

**Review by mathis-hermann — CHANGES_REQUESTED** [review]
@vinothss4u Thanks for the PR. Please see the two comments.

File: payrexx/src/Service/PayrexxOrderService.php [hunk 2]
```diff
@@ -27,6 +27,10 @@ class PayrexxOrderService
     // ID 10
     const PS_STATUS_BANKWIRE = 'PS_OS_BANKWIRE';
```
[the remark this answers is not in the document]
Comment (vinothss4u) [reply payrexx/src/Service/PayrexxOrderService.php:29]: Updated the ID

File: payrexx/src/Service/PayrexxOrderService.php (hunk 1 above)
Comment (vinothss4u) [reply payrexx/src/Service/PayrexxOrderService.php:116]: @mathis-hermann 

The PS_OS_BANKWIRE(10) text is Awaiting bank wire payment

It is not a final state.

Comment (mathis-hermann) [reply payrexx/src/Service/PayrexxOrderService.php:116]: @vinothss4u Ok - but then this case ` return $oldStatusId !== (int) \Configuration::get(self::PS_STATUS_BANKWIRE);` (old line 109) is missing?

**Review by mathis-hermann — APPROVED** [review]
@vinothss4u Thanks for the PR. Please make a release.

**Pull request merged**